docs/source/tutorials/projects.rst

```diff
@@ -12,19 +12,31 @@ Make sure that you have associated a :xref:`public-private_key_pair` with |compa
 If not you can do so by following the :ref:`authentication:User authentication` tutorial.
 
 Creating a Project
-==================
+====================
```

Comment:
I have a feeling we should deprecate most of this given the autodoc that we have and just point to the autodoc. It seems a bit weird to me to have the same thing documented in multiple places.